README.md

```diff
@@ -11,7 +11,6 @@ Created by [@jamiepine]() | Sponsored and maintained by the [Notify Team]()
 <p align="left">
   <a href="https://discord.gg/RjG8ShB"><img src="https://discordapp.com/api/guilds/658189217746255881/embed.png" alt="Join Discord"></a>
   <a href="https://patreon.com/jamiepine"><img src="https://img.shields.io/badge/donate-patreon-F96854.svg" alt="Donate on patreon"></a>
-  <a href="https://twitter.com/jamiepine"><img src="https://img.shields.io/twitter/follow/jamiepine.svg?label=Jamie's Twitter" alt="Follow on twitter"></a>
```

Comment:
Since Pulse is primarily Jamie's project, I don't think we should remove this link. However, the Patreon page doesn't exist anymore and should probably be removed. Would you mind making this change?

Reply:
Sure thing! Yeah, I only removed it because the image URL was not functioning properly. That, however, can be remedied. :) And yes, I'll remove Patreon. :)